modoboa/extensions/stats/management/commands/logparser.py

```diff
@@ -297,6 +297,14 @@ def _parse_line(self, line):
         if not m:
             return
         host, prog, pid, log = m.groups()
+        m = re.search("(INFECTED|SPAMMY) .* <[^>]+> -> <[^@]+@([^>]+)>.*", log)
```

Comment:
Sorry, my question was unclear. In addition to the keywords you already use, there is another one which is ```SPAM```. I just wonder why you don't catch it.